tests/Unit/Domain/CoordinateMaps/TimeDependent/Test_RotScaleTrans.cpp

```diff
@@ -93,89 +96,121 @@ void test_RotScaleTrans() {
-  scale_trans_map_non_rigid =
-      serialize_and_deserialize(scale_trans_map_non_rigid);
+      scale_map_transition = map_serialize_and_deserialize(
+          scale_pair, std::nullopt, std::nullopt, BlockRegion::Transition);
+
+  // Scaling Outer
+  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> scale_map_outer =
+      map_serialize_and_deserialize(scale_pair, std::nullopt, std::nullopt,
+                                    BlockRegion::Outer);
+
+  // Translation Inner
+  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> trans_map_inner =
+      map_serialize_and_deserialize(std::nullopt, std::nullopt, "translation",
+                                    BlockRegion::Inner);
 
-  // Scaling, Translation Rigid
+  // Translation Transition
   domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim>
-      scale_trans_map_rigid{scale_pair,   std::nullopt, "translation",
-                            inner_radius, outer_radius, true};
-  scale_trans_map_rigid = serialize_and_deserialize(scale_trans_map_rigid);
-
-  // Rotation
-  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> rot_map{
-      std::nullopt, "rotation_angle", std::nullopt,
-      inner_radius, outer_radius,     true};
-  rot_map = serialize_and_deserialize(rot_map);
-
-  // Scaling Non-Rigid
-  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> scale_map_non_rigid{
-      scale_pair,   std::nullopt, std::nullopt,
-      inner_radius, outer_radius, false};
-  scale_map_non_rigid = serialize_and_deserialize(scale_map_non_rigid);
-
-  // Scaling Rigid
-  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> scale_map_rigid{
-      scale_pair, std::nullopt, std::nullopt, inner_radius, outer_radius, true};
-  scale_map_rigid = serialize_and_deserialize(scale_map_rigid);
-
-  // Translation Non-Rigid
-  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> trans_map_non_rigid{
-      std::nullopt, std::nullopt, "translation",
-      inner_radius, outer_radius, false};
-  trans_map_non_rigid = serialize_and_deserialize(trans_map_non_rigid);
-
-  // Translation Rigid
-  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> trans_map_rigid{
-      std::nullopt, std::nullopt, "translation",
-      inner_radius, outer_radius, true};
-  trans_map_rigid = serialize_and_deserialize(trans_map_rigid);
-
-  UniformCustomDistribution<double> dist_double{-5.0, 5.0};
-  UniformCustomDistribution<double> far_dist_double{50.0, 60.0};
+      trans_map_transition = map_serialize_and_deserialize(
+          std::nullopt, std::nullopt, "translation", BlockRegion::Transition);
+
+  // Translation Outer
+  domain::CoordinateMaps::TimeDependent::RotScaleTrans<Dim> trans_map_outer =
+      map_serialize_and_deserialize(std::nullopt, std::nullopt, "translation",
+                                    BlockRegion::Outer);
+
+  double far_double_1 = 0;
+  double far_double_2 = 0;
+  if (Dim == 2) {
+    far_double_1 = 30.0;
+    far_double_2 = 35.36;
+  } else {
```

Comment:
Could you explain why this is necessary? I'm not sure I understand.

Also, if we keep it, these can be one-liners
```cpp
const double far_double_1 = Dim == 2 ? 30.0 : 25.0;
const double far_double_2 = Dim == 2 ? 35.36 : 28.87;
```

Reply:
I wanted points that were pretty close to the outer boundary, but not too far beyond. This helped fix the roundoff errors for the jacobian tests. After some more testing though, the frame velocity rame velocity still has trouble reaching 1e-10 super consistently though when rotation is specified so the quaternion is still causing some trouble.

Reply:
But the outer radius is `50` from above?

Reply:
Oh maybe my variable naming is confusing, these numbers go in as the far distribution for far_point_xi. So it gives far_point_xi 2 numbers between 30 and 35 in 2D so that when you take the magnitude of it it's pretty close to 50 and similar for 3D

Reply:
Ohhhh. Ok I got it. If you want to have more control over the radius, I suggest using a random distribution from 0 to 50 (or a bit outside) for the radius, and a random dist for the azimuthal angle. Then for 3D you can have another random dist for the polar angle. That way you don't have to guess for the x/y/z components..